src/ex1.py

```diff
@@ -0,0 +1,20 @@
+def get_week_day (week: int) -> int:
+
+    day = ""
+
+    if week == 1:
```

Comment:
sarebbe il caso di controllare se week non ha un valore nel range (1-7), cioè se il valore che ci stanno fornendo in input non è tra quelli attesi, ed in questo caso ritornare None per fa capire a chi usa la funzione che qualcosa è andato storto